setup/options.rst

```diff
@@ -0,0 +1,228 @@
+Options
+=======
+
+There are a number of options for OpenRCT2 that can be tuned to fit your preferences and needs. They are broken up into a number of different sections.
```

Comment:
Include how to open the window. You can use the staff page for inspiration.

(Make sure to screenshot both the button and the menu option.)